server/src/main/java/org/apache/uniffle/server/storage/MultiStorageManager.java

```diff
@@ -65,39 +73,51 @@ public Storage selectStorage(ShuffleDataReadEvent event) {
 
   @Override
   public void updateWriteMetrics(ShuffleDataFlushEvent event, long writeTime) {
-    selectStorageManager(event).updateWriteMetrics(event, writeTime);
+    throw new UnsupportedOperationException();
```

Comment:
Why do we throw this exception?

Reply:
1.`org.apache.uniffle.server.storage.MultiStorageManager#updateWriteMetrics` is not referenced, so it is useless.
2.Cache entry for this event will be invalidated after write success

Reply:
Ok, I got it.